res/controllers/Hercules-DJ-Console-RMX-scripts.js

```diff
@@ -909,7 +909,7 @@ HerculesRMX.rateChange = function (value, group) {
 //   HerculesRMX.stopJog[group] = 1.0;
 //   HerculesRMX.decayLast = new Date().getTime();
 //   engine.setValue(group,"scratch",HerculesRMX.stopJog[group]);
-//   HerculesRMX.stoping = true;
+//   HerculesRMX.stopping = true;
```

Comment:
i couldn't find `.stoping` in the rest of the source or `.stopping` for that matter either. I just corrected this in order to ask what to do about this.

Reply:
this is just a comment (`//`) so correcting it should be no problem.